extensions/interactions/NumericInput/protractor.js

```diff
@@ -31,7 +31,8 @@ var expectInteractionDetailsToMatch = function(elem) {
 
 var submitAnswer = function(elem, answer) {
   elem.element(by.tagName('oppia-interactive-numeric-input')).
-    element(by.tagName('input')).sendKeys(answer + '\n');
+    element(by.tagName('input')).sendKeys(answer);
```

Comment:
Why this change?

Reply:
Hi, I've mentioned above this change in the testing doc [here](https://docs.google.com/document/d/1ZRzKY3MCUpOZYoDj2ISELv_sEYiiOuOZkE-8MMjUQuY/edit?usp=sharing) md has removed many action events because of its co-existence
/n is an alternative for submitting/clicking of an input  e2e tests.
as the event actions are removed /n won't work in md-inputs